CMakeLists.txt

```diff
@@ -64,14 +64,29 @@ target_link_libraries(${BIN_TARGET} PRIVATE
 )
 
 set(TARGET_PLATFORM host CACHE STRING "Target platform")
-set_property(CACHE TARGET_PLATFORM PROPERTY STRINGS host retrofw gcw0 rg350)
+set_property(CACHE TARGET_PLATFORM PROPERTY STRINGS host retrofw gcw0 rg350 miyoo)
 
 if(TARGET_PLATFORM STREQUAL "retrofw")
   include(CMake/retrofw_defs.cmake)
 elseif(TARGET_PLATFORM STREQUAL "gcw0")
   include(CMake/rg350_defs.cmake)
 elseif(TARGET_PLATFORM STREQUAL "rg350")
   include(CMake/rg350_defs.cmake)
+elseif(TARGET_PLATFORM STREQUAL "miyoo")
+  include(CMake/miyoo_defs.cmake)
+  target_link_libraries(${BIN_TARGET} PRIVATE
+  mpg123
```

Comment:
Strange that these explicit link targets are needed on Miyoo but nowhere else

Reply:
It is not strange. The toolchain for Miyoo builds statically, so all dependencies need to be given at compile time.

Reply:
I'd expect the pkgconfig / cmake config files for these static deps to specify the link requirements corectly (in which case this shouldn't be needed) but I guess they don't.

Reply:
True, I did not think about it earlier. Anyway, it is a sufficient workaround so let it be.